arches/app/search/search_export.py

```diff
@@ -314,7 +321,10 @@ def flatten_tiles(self, tiles, datatype_factory, compact=True, use_fieldname=Fal
                 node = self.get_node(nodeid)
                 if node.exportable:
                     datatype = datatype_factory.get_instance(node.datatype)
-                    node_value = datatype.get_display_value(tile, node)
+                    if 'concept' in node.datatype and self.concept_uuid_format:
+                        node_value = datatype.transform_export_values(value, concept_export_value_type='id')
+                    else:
+                        node_value = datatype.get_display_value(tile, node)
```

Comment:
This could just be: 

```
if self.export_system_value:
    node_value = datatype.transform_export_values(value, **{"concept_export_value_type": "id"})
else:
    node_value = datatype.get_display_value(tile, node)
```

Reply:
fix in latest